src/pyrokinetics/local_geometry/metric.py

```diff
@@ -130,17 +128,19 @@ def __init__(self, local_geometry: LocalGeometry, ntheta=None, theta=None):
 
         # poloidal average of Jacobian * g^zetazeta: <Jacobian * g^zetazeta>,
         # e.g. the denominator of equation 16
-        self.Y = (
-            integrate.trapezoid(self.Jacobian / self.R**2, self.regulartheta)
-            / self.theta_range
+        self.Y = integrate.trapezoid(self.Jacobian / self.R**2, self.regulartheta) / (
+            2 * np.pi
```

Comment:
Not sure there's a harm in it but @hgd511 was keen on keeping it between -pi to pi. There's no harm in increasing it though unless I've missed something

Reply:
I think keeping the theta domain for the metric terms between -pi to pi makes equation 155 easier to understand, as one can clearly see the role of kx (~theta0) and the number of 'turns' (~2 M pi, for Mth turn). It also works for when one interprets the flux tube geometry as a single turn and multiple kx, i.e M = 0 and kx = 2 pi nx / Lx. Happy to discuss if there's a better way of doing it!

Could be worth keeping theta_range as a quantity for plotting against ballooning angle (although I assume this already exists for eigenfunctions etc).

Reply:
I was just thinking that given that a flux tube "cell" doesn't have to be 2 pi in length it might help comparisons to codes if the metric calculation can also be constructed from these "extended cells".

Reply:
I can add back in the range but leave the default as from -pi to pi